framework/src/controller/schema/application_config_schema.js

```diff
@@ -159,12 +164,12 @@ module.exports = {
 								},
 								OFFSET: {
 									type: 'integer',
-									min: 1,
+									minimum: 1,
```

Comment:
For future reference, here is the specification for this change. 

https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.2.4